From: Markus Armbruster <armbru@redhat.com>
Date: Fri, 15 Aug 2008 18:16:25 -0400
Subject: [xen] PVFB probe & suspend fixes fix
Message-id: m33al59b5y.fsf_-_@crossbow.pond.sub.org
O-Subject: [PATCH RHEL-5.3] Xen PVFB probe & suspend fixes fix
Bugzilla: 459107
RH-Acked-by: Arnaldo Carvalho de Melo <acme@redhat.com>
RH-Acked-by: Bill Burns <bburns@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>

An embarrassing typo in linux-2.6-xen-pvfb-probe-suspend-fixes.patch
escaped review.

Instead of committing this as a separate patch, we could simply update
the flawed patch.  Matter of taste.

Bug 459107.  Please ACK.

diff --git a/drivers/xen/fbfront/xenkbd.c b/drivers/xen/fbfront/xenkbd.c
index cb409ed..1dfa88a 100644
--- a/drivers/xen/fbfront/xenkbd.c
+++ b/drivers/xen/fbfront/xenkbd.c
@@ -103,7 +103,7 @@ int __devinit xenkbd_probe(struct xenbus_device *dev,
 	info->xbdev = dev;
 	info->irq = -1;
 
-	info->page = (void *)__get_free_page(GFP_KERNEL || __GFP_ZERO);
+	info->page = (void *)__get_free_page(GFP_KERNEL | __GFP_ZERO);
 	if (!info->page)
 		goto error_nomem;
 
